install_config/configuring_aws.adoc

```diff
@@ -166,6 +166,66 @@ AWS_SECRET_ACCESS_KEY=<secret_key>
 Access keys are obtained when setting up your AWS IAM user.
 ====
 
+[[aws-cluster-labeling]]
+== Labeling Clusters for AWS
+This topic describes how to label an existing {product-title} cluster running on
+Amazon Web Services (AWS).
+
+To correctly identify which resources are associated with a cluster, tag
+resources with the key `kubernetes.io/cluster/<name>`, where `<name>` is a unique
+name for the cluster. Tagging all resources with the
+`kubernetes.io/cluster/<name>` tag avoids potential issues with multiple zones
+or multiple clusters.
+
+[NOTE]
```

Comment:
@gnunn1 I have added this note which includes `clusterid` info. Does this looks alright?

Reply:
I am also trying to get more info about `openshift_clusterid` variable meanwhile.